code/style.css

```diff
@@ -5,3 +5,368 @@ this can be used for the FAQ-accordion. read more here: https://www.w3schools.co
 .active + .answer {
   /* some styling here */
 }
+
+body {
+margin: 0;
+color: #888888;
+
+}
+
+
+/*style the header*/
+.header{
+  width: 100%;
+  height: 40vh;
+  overflow: hidden;
+
+}
+
+
+.header h1 {
+  color: #D8E8FF;
+  text-shadow: 2px 2px 5px #888888;
+ font-size: 70px;
+
+
+}
+
+/** button on the header**/
+.knapp {
+  height: 80px;
+  width: 80px;
+  border-radius: 50%;
+  background-color:transparent;
+  box-shadow: inset 0 0 10px white;
+  border:none;
+}
+
+.knapp:active {
+  transform: translateY(4px);
+  box-shadow: 5px 5px 10px grey;
+}
+
+.knapp:focus {
+  outline: none;
+}
+
+.active, .knapp:hover {
+  background: white;
+  opacity: 50%;
+}
+
+/** Hero Video **/
+
+.header .header__video{
+  object-fit: cover;
+  width: 100%;
+  height: 100%;
+}
+ 
+.overlay {
+  position: absolute;
+  top: 0px;
+ width: 100%;
+ height:40%;
+ text-align: center;
+}
+
+/** Content area **/
+
+.content {
+  height: 1250px;
+  background: linear-gradient(#D8E8FF,#CEAEA5);  
+
+ font-family: 'Josefin Sans', sans-serif;
+  font-family: 'Lato', sans-serif;
```

Comment:
Having two font-family properties like this only makes the last one valid, since the computer read the code from top to bottom with the most recent being valid.